connaisseur/tests/test_mutate.py

```diff
@@ -18,6 +18,22 @@
         ]
     },
 }
+request_obj_pod_with_init_container = {
+    "kind": "Pod",
+    "apiVersion": "v1",
+    "metadata": {},
+    "spec": {
+        "containers": [
+            {"name": "test-connaisseur", "image": "phbelitz/charlie-image:test"}
```

Comment:
I'd say similarly to moving the demo images to sse, we should also move these test images to sse repo, but might be out of scope here

Reply:
Good point! However, at least for the tests I added, it doesn't fully apply as this just parses the values, so I could use any name I want and the repo would not even need to exist. We don't have test images in the sse repo yet, do we?

Reply:
Nope, added comment to #41 and will resolve here